device/cameo/x86_64-cameo_esc600_128q-r0/plugins/eeprom.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
```

Comment:
These old platform plugins (eeprom.py, psuutil.py, sfputil.py) are deprecated in favor of the new sonic-platform package. Support for these will be removed soon in the master branch. I will allow these in the 201911 branch, but for master you should generate a Python 3 sonic-platform wheel package instead.